repose-aggregator/commons/commons-utilities/src/test/groovy/org/openrepose/commons/utils/http/ServiceClientTest.groovy

```diff
@@ -75,9 +77,26 @@ class ServiceClientTest {
         serviceClient.post("http://example.com", headers, "body", MediaType.TEXT_PLAIN_TYPE)
 
         verify(mockHttpClient).execute(any(HttpRequestBase))
-        assertTrue(requestCaptor.getValue().containsHeader("x-foo"))
-        assertTrue(requestCaptor.getValue().containsHeader("x-bar"))
-        assertThat(requestCaptor.getValue().getFirstHeader("x-foo").getValue(), equalTo("foo"))
-        assertThat(requestCaptor.getValue().getFirstHeader("x-bar").getValue(), equalTo("bar"))
+        assertThat(requestCaptor.getValue().getFirstHeader("x-foo"), hasValue("foo"))
+        assertThat(requestCaptor.getValue().getFirstHeader("x-bar"), hasValue("bar"))
+    }
+
+    Matcher<Header> hasValue(String value) {
+        return new TypeSafeMatcher<Header>() {
+
+            @Override
+            boolean matchesSafely(Header item) {
+                boolean result = false
+                if (item != null) {
+                    result = item.getValue() == value
```

Comment:
There's the winner. I have no idea why I couldn't come up with that yesterday. I am ashamed.